pt-br/tutorials/tour/implicit-parameters.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: tutorial
+title: Parâmetros Implícitos
+
+disqus: true
+
+tutorial: scala-tour
+num: 25
+tutorial-next: implicit-conversions
+tutorial-previous: explicitly-typed-self-references
+language: pt-br
+---
+
+Um método com _parâmetros implícitos_ pode ser aplicado a argumentos como um método normal. Neste caso, o rótulo implícito não tem efeito. No entanto, se esse método faltar argumentos para os parâmetros implícitos declarados, tais argumentos serão automaticamente fornecidos.
+
+Os argumentos reais que são elegíveis para serem passados para um parâmetro implícito se dividem em duas categorias:
+
+* Primeira, são elegíveis todos os identificadores x que podem ser acessados no ponto da chamada do método sem um prefixo e que denotam uma definição implícita ou um parâmetro implícito.
+
+* Segunda, são elegíveis também todos os membros dos módulos acompanhates do tipo do parâmetro implícito que são rotulados como `implicit`.
+
+No exemplo a seguir, definimos um método `sum` que calcula a soma de uma lista de elementos usando as operações `add` e `unit` do monoide. Observe que valores implícitos não podem ser de nível superior, eles precisam ser membros de um modelo.
```

Comment:
Is `nivel superior` a known translation to `top-level` in portuguese? If not, maybe put the english expression in parenthesis? Or just use the original? I'm saying this because I needed to see the original text to understand `nivel superior` meaning.